designer/server/src/views/file/download-page.njk

```diff
@@ -0,0 +1,37 @@
+{% extends 'layouts/page.njk' %}
+
+{% from "govuk/components/back-link/macro.njk" import govukBackLink %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "govuk/components/input/macro.njk" import govukInput %}
+
+{% block content %}
+  <form method="post" novalidate>
+    {% call appPageBody({
```

Comment:
Can you indent everything inside this call block by 2 spaces to match **download-complete.njk**?

Reply:
done